app/scripts/modules/core/src/pipeline/config/stages/googleCloudBuild/GoogleCloudBuildStageForm.tsx

```diff
@@ -144,6 +195,34 @@ export class GoogleCloudBuildStageForm extends React.Component<
             updatePipeline={this.props.updatePipeline}
           />
         )}
+        {stage.buildDefinitionSource === buildDefinitionSources.TRIGGER && (
+          <React.Fragment>
+            <StageConfigField label="Trigger Name">
+              <Select
+                clearable={false}
+                onChange={this.onTriggerChange}
+                options={this.getCloudBuildTriggers()}
+                value={stage.triggerId}
+              />
+            </StageConfigField>
+            <StageConfigField label="Trigger Type">
+              <Select
+                clearable={false}
+                onChange={this.onTriggerTypeChange}
+                options={this.getTriggerTypes()}
+                value={stage.triggerType}
+              />
+            </StageConfigField>
+            <StageConfigField label="Value">
+              <TextInput
+                type="text"
+                className="form-control"
+                onChange={this.onTriggerTypeValueChange}
```

Comment:
```suggestion
                onChange={this.onTriggerValueChange}
```

Might be easier to read if we remove `type` from the name of the methods related to the value?